filters/json/src/test/java/com/spartansoftwareinc/ws/okapi/filters/json/JSONFilterConfigurationDataTest.java

```diff
@@ -12,14 +12,15 @@
 
 public class JSONFilterConfigurationDataTest {
 
-    private final static String TERMINATOR = System.getProperty("line.separator");
+    private final static String TERMINATOR = "\n";
```

Comment:
@denis-konovalyenko does this change not cause problems for you?  The test originally used `\n`, but this was switched to `line.separator` in commit 10d8127f because the test was failing on Windows.